sdk/trace/batch_span_processor_test.go

```diff
@@ -201,4 +201,20 @@ func getSpanContext() core.SpanContext {
 		SpanID:     sid,
 		TraceFlags: 0x1,
 	}
-}
\ No newline at end of file
+}
+
+func TestBatchSpanProcessorShutdown(t *testing.T) {
+	bsp, err := sdktrace.NewBatchSpanProcessor(&testBatchExporter{})
+	if err != nil {
+		t.Errorf("Unexpected error while creating processor\n")
+	}
+
+	if bsp == nil {
+		t.Fatalf("Error creating new instance of BatchSpanProcessor\n")
+	}
+
+	bsp.Shutdown()
+
+	// Check if it only executes once
```

Comment:
nit: Since there is no actual check here maybe rephrase the comment to "Multiple call to Shutdown() should not panic."

Reply:
Thank you, I fixed the comment.